server/sonar-server/src/main/java/org/sonar/server/platform/web/RoutesFilter.java

```diff
@@ -51,6 +54,22 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
     } else if (API_SOURCES_WS.equals(path)) {
       // SONAR-7852 /api/sources?resource url is still used
       response.sendRedirect(format("%s%s/index?%s", request.getContextPath(), API_SOURCES_WS, request.getQueryString()));
+    } else if (!path.equals("/api/properties/index") && path.startsWith("/api/properties")) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "/api/properties" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the "/api/properties/index" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1132)